funcx_endpoint/funcx_endpoint/endpoint/rabbit_mq/result_queue_subscriber.py

```diff
@@ -16,10 +18,17 @@ class ResultQueueSubscriber(multiprocessing.Process):
 
     """
 
+    QUEUE_NAME = "results"
+    ROUTING_KEY = "*.results"
+    EXCHANGE_NAME = "results"
+    EXCHANGE_TYPE = "topic"
```

Comment:
Will these always precisely match the values of `ResultQueuePublisher`?  If so, seems a shame to have multiple definitions lying around.

Reply:
It might be nice to either put these in a little helper or do something else with them.
I advocated in #701 to move these from instance attributes to class attributes to improve readability -- the reason being that they are class-based constants which we could tune.

An underlying thought: we could adjust several of these with a testing-specific subclass, so this allows a kind of handy reflection in which we write:
```python
class CustomRQSubscriber(ResultQueueSubscriber):
    QUEUE_NAME = f"results-{uuid.uuid1()}"
```
or whatever turns out to be the "right way" to parametrize this so that it can be run by multiple test runs in parallel without conflicts.

Reply:
I may have buried the lede there a bit. My thought is that if we made it somehow shared, I still want to be able to customize it in tests, so I'm refraining from making another change for now.

Possibly it could be a container of some kind, like
```
@dataclasses.dataclass
class ResultQueueSettings:
    queue_name: str = dataclasses.field(default="results")
    ...

class ResultQueueSubscriber:
    settings = ResultQueueSettings()
```

But I haven't noodled that all out yet.